pkg/TIPA.py

```
@@ -72,63 +72,61 @@ def run(self):
                 break
-            keyboard.on_press_key(key="f", callback=self.on_release)
-            time.sleep(.1)
-            self.listen_lock = False
 
-        # # Listener Loop
-        # with pkeyboard.Listener(on_press=lambda var:self.on_press(var)) as listener:
-        #     listener.join()
-
-    def Close(self):
-        print("\nStopping")
+    def close(self):
```

Ruff (0.8.0)

Missing return type annotation for public function `close`

Add return type annotation: `None`

(ANN201)